translations/zh-CN.json

```
@@ -989,22 +989,22 @@
 	"Unable to pick a default driver. Here is what was considered, in preference order:": "",
 	"Unable to pull images, which may be OK: {{.error}}": "无法拉取镜像，有可能是正常状况：{{.error}}",
 	"Unable to push cached images: {{.error}}": "",
-	"Unable to remove machine directory": "",
+	"Unable to remove machine directory": "无法删除计算机目录",
```

How to translate this machine is really a tricky thing. But it is definitely not"computer(计算机)". The "machine" here is actually the place where minikube is deployed（like a docker container or a vm）, and actually its name comes from the library's original name "Docker Machine". https://github.com/docker/machine

machine这东西怎么翻译真挺棘手的，但是他肯定不是计算机的意思。machine这玩意本质上代指的是跑minikube的那个虚拟环境，他可能是个docker 容器，也可能是个虚拟机啥的。这东西之所以叫machine是因为那个操作这些虚拟化环境的库，最最开始名字叫做docker machine https://github.com/docker/machine 。所以要我说不如不翻译了算了，这个词直接扔这吧。

So I suggest that we should just leave this word here. 

```suggestion
	"Unable to remove machine directory": "无法删除machine目录",
```